content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -365,9 +368,43 @@ poorly-behaved workloads that may be harming system health.
   long requests took to actually execute, grouped by the FlowSchema that matched the
   request and the PriorityLevel to which it was assigned.
 
-
-
-
+### Debug endpoints
+
+When you enable the API Priority and Fairness feature, the kube-apiserver serves the following additional paths at its HTTP[S] ports.
+
+- `/debug/api_priority_and_fairness/dump_priority_levels` - a listing of all the priority levels and the current state of each.  Following is an example.
+  ```
+  $ kubectl get --raw /debug/api_priority_and_fairness/dump_priority_levels
```

Comment:
nit: it's better to [separate commands from their output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output), and that also lets you turn on highlighting for the shell part.